opencti-platform/opencti-graphql/src/database/rabbitmq.js

```diff
@@ -320,9 +320,7 @@ export const consumeQueue = async (context, connectorId, connectionSetterCallbac
         } else { // Connection success
           logApp.info('[QUEUEING] Starting connector queue consuming', { connectorId });
           conn.on('close', (onConnectError) => {
-            if (onConnectError) {
-              reject(onConnectError);
-            }
+            reject(onConnectError);
```

Comment:
Is there a reason to change this? I remember that it was intended this way: close event can be sent for regular connection close, it doesn't always mean that there is an error

Reply:
not sure on this change I think I inherited it from https://github.com/OpenCTI-Platform/opencti/pull/7594